SU-HSDO/suhumsci
# HSD8-681 prune courses

# READY FOR REVIEW

# Summary
- Delete orphans of courses migration.

# Needed By (Date)
- 7/31

# Urgency
- medium

# Steps to Test
1. checkout this branch
1. `composer install --prefer-source`
1. run a fresh install `drush si config_installer -y; drush cim -y; drush uli`
1. on `/admin/structure/migrate/course-importer` set the course importer to a really small set of courses like `https://explorecourses.stanford.edu/search?q=MS%26E111&view=xml-20140630` 
1. run migration `drush mim hs_courses`
1. view `/admin/content` and count and take note of what courses have imported
1. back on `/admin/structure/migrate/course-importer` change the importer to another set of courses, something like `https://explorecourses.stanford.edu/search?q=MS%26E112&view=xml-20140630`
1. run migration `drush mim hs_courses`
1. view `/admin/content` and verify the original courses have been removed and you now only have courses from the new xml source.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
